regexes/oss.yml

```diff
@@ -421,7 +421,7 @@
   name: 'iOS'
   version: '$1'
 
-- regex: 'Podcasts/(?:[\d\.]+)|Instacast(?:HD)?/(?:\d\.[\d\.abc]+)|Pocket Casts, iOS|Overcast|Castro|Podcat|i[cC]atcher|RSSRadio/'
+- regex: 'Podcasts/(?:[\d\.]+)|Instacast(?:HD)?/(?:\d\.[\d\.abc]+)|Pocket Casts, iOS|Overcast|Castro|Podcat|Procast|i[cC]atcher|RSSRadio/'
```

Comment:
Guess that's not correct, as the app is also available for Android: https://play.google.com/store/apps/details?id=com.podflitzer.android